queries/tokenLists/use1InchTokenList.ts

```diff
@@ -0,0 +1,44 @@
+import { useQuery, QueryConfig } from 'react-query';
+import axios from 'axios';
+import keyBy from 'lodash/keyBy';
+
+import QUERY_KEYS from 'constants/queryKeys';
+import { CRYPTO_CURRENCY_MAP, ETH_ADDRESS } from 'constants/currency';
+
+import { TokenListQueryResponse, TokenListResponse } from './types';
+
+const ether = {
+	address: ETH_ADDRESS,
+	chainId: 1,
+	decimals: 18,
+	logoURI:
+		'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/info/logo.png',
+	name: 'Ethereum',
+	symbol: CRYPTO_CURRENCY_MAP.ETH,
+	tags: [],
+};
```

Comment:
The reason I'm adding `ether` as a "token" here is just for convenience, so that we can easily use it within the `SelectTokenModal` as if it was just another token.